code/+sitools/@ai_recorder/ai_recorder.m

```diff
@@ -183,16 +183,87 @@
             % ai_recorder.connectToDAQ - Connect to the DAQ using the object properties
             %
             % Purpose
-            % Run this method to connect to an NI DAQ device for analog
+            % Run this method to connect to an DAQ device for analog
             % input using the parameters described in the properties of
             % this class. i.e.
-            % devName - the name of the NI device
+            % devName - the name of the device
```

Comment:
Should probably put in the docs how to select vDAQ or NI DAQ? Or add examples in doc text?

Reply:
On my setup, the device name starts with v but it shouldn't matter as long as it matches what scanimage calls it.